components/TForm.vue

```diff
@@ -157,8 +157,6 @@ export default {
       if (field && field.onChange) {
         field.onChange(val)
       }
-
-      this.$emit('input', val)
```

Comment:
this is probably used somewhere, did you check?

Reply:
i did and form seems to function the same without it

Reply:
check commit where this line was added and see why

Reply:
It sets a new value for the form, whenever any field is changed. If this event is not emitted `v-model` of the form won't be updated.